client/src/components/App/Content/InvestigationForm/TabManagement/TabManagement.tsx

```diff
@@ -1,15 +1,42 @@
 import React from 'react';
-import {Paper, Tabs, Tab, Card, withStyles, createStyles} from '@material-ui/core';
+import rtl from 'jss-rtl';
+import { create } from 'jss';
+import { jssPreset } from '@material-ui/styles';
+import {Paper, Tabs, Tab, Card, StylesProvider, createStyles, withStyles} from '@material-ui/core';
 
 import TabObj from 'models/TabObj';
```

Comment:
You can call it Tab instead of TabObj and than use Tab as TabObj to prevent duplicate names